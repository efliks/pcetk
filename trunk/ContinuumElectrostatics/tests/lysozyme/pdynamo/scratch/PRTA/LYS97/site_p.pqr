ATOM      0  CE  LYS    97      -6.724  16.480  32.153   0.21000   1.700      0      0      0      0
ATOM      0  HE1 LYS    97      -7.421  15.927  31.507   0.05000   1.000      0      0      0      0
ATOM      0  HE2 LYS    97      -7.133  17.407  32.580   0.05000   1.000      0      0      0      0
ATOM      0  NZ  LYS    97      -6.122  15.643  33.146  -0.30000   1.550      0      0      0      0
ATOM      0  HZ1 LYS    97      -5.476  16.279  33.691   0.33000   0.300      0      0      0      0
ATOM      0  HZ2 LYS    97      -5.503  14.880  32.761   0.33000   0.300      0      0      0      0
ATOM      0  HZ3 LYS    97      -6.807  15.233  33.802   0.33000   0.300      0      0      0      0
