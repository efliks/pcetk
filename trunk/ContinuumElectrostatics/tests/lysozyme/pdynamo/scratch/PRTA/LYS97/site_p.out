Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/LYS97//site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.664701, 20.5141, 19.0512)) 121 2
Finer level:    (-6.45524, 15.9784, 32.8057) 101 1
Finer level:    (-6.45524, 15.9784, 32.8057) 101 0.5
Finest level:   (-6.45524, 15.9784, 32.8057) 101 0.25


Self energy of scratch/PRTA/LYS97//site_p in three dielectric environment = 104.52
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/LYS97//site_p with background charge set in three dielectric environment = -0.261134
(probably in kcal/mole)



Interaction energies of scratch/PRTA/LYS97//site_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.0118366
0   1 : -0.000115988
1   0 : 0.000529165
1   1 : -0.00873301
2   0 : 0.0181386
2   1 : 0.000526097
3   0 : 0.082603
3   1 : -0.000461863
3   2 : 0.00681275
3   3 : -0.0703007
4   0 : 0.0054155
4   1 : -0.0212482
5   0 : 0.0200036
5   1 : -0.347883
6   0 : 0.000924807
6   1 : -0.0426903
7   0 : 0.00305853
7   1 : 9.87656e-05
8   0 : 0.00282545
8   1 : -0.0303431
9   0 : -0.00172727
9   1 : -0.0245468
10   0 : -0.000168559
10   1 : -0.0388004
11   0 : 0.00305679
11   1 : -0.0318003
12   0 : 0.00301812
12   1 : -0.0396584
13   0 : 0.00104604
13   1 : -0.0536335
14   0 : 0.245041
14   1 : 8.07622e-05
15   0 : 208.99
15   1 : 55.7292
16   0 : -0.0541265
16   1 : -0.528928
17   0 : 0.0154463
17   1 : -0.000141345
18   0 : 0.000109409
18   1 : -0.00281395
Total runtime: 16.23 s user 0.13 s system 
